ui/src/app/certificate_requests/row.tsx

```diff
@@ -0,0 +1,32 @@
+const extractCSR = (csrPemString: string) => {
+    //TODO
+}
+
+const extractCert = (certPemString: string) => {
+    //TODO
+}
+
+type rowProps = {
+    id: number,
+    csr: string,
+    certificate: string
+}
+export default function Row({ id, csr, certificate }: rowProps) {
+    return (
+        <tr>
+            <td className="" data-test-column="id">{id}</td>
+            <td className="u-overflow--visible" data-test-column="details">
+                <p>CN: example.com</p>
```

Comment:
Let's avoid accronyms when possible, we can use Common Name instead of CN. Same comment for SAN

Reply:
Changed